pyproject.toml

```diff
@@ -46,6 +46,7 @@ jax = ">=0.3"
 jupyter = {version = ">=1.0", optional = true}
 leidenalg = {version = "*", optional = true}
 loompy = {version = ">=3.0.6", optional = true}
+mudata = { git = "https://github.com/scverse/mudata.git", rev = "c3bf23c" }
```

Comment:
need an official versioned release

Reply:
none of the official ones work, should i just ping them to release?